src/test/java/com/hubspot/smtp/messages/DotStuffingChunkedStreamTest.java

```diff
@@ -0,0 +1,62 @@
+package com.hubspot.smtp.messages;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
nit: I like to use assertJ rather than assertEquals. It makes for better error message. Plus, I think some of your tests would really benefit from softassertions:

```java
SoftAssertions softly = new SoftAssertions();

softly.assertThat(dotStuff(".", chunkSize)).isEqualTo(".." + CRLF);
softly.assertThat(dotStuf(".abc", chunkSize)).isEqualTo("..abc");
// ...

softly.assertAll();
```